test/test_helper.rb

```diff
@@ -2,3 +2,4 @@
 SimpleCov.start
 require 'minitest/autorun'
 require 'minitest/pride'
+require 'mocha/minitest'
```

Comment:
Thanks for adding the "mocha" library here. Definitely think we'll need it in the future for mocks and stubs.